RustFxA/RustFirefoxAccounts.swift

```diff
@@ -8,19 +8,25 @@ import SwiftKeychainWrapper
 
 fileprivate let prefs = NSUserDefaultsPrefs(prefix: "profile")
 
+/**
+ A singleton that wraps the Rust FxA library.
+ The singleton design is poor for testability through dependency injection and may need to be changed in future.
+ To be renamed FirefoxAccounts.swift once the old code is removed fully.
```

Comment:
Should we put a Todo for renaming but like we discussed a while ago that even those are are bad 😅 (either way is fine)